File: 08.AndroidBasic/Chap7/SampleNinePatch/app/src/main/java/org/techtown/ninepatch/MainActivity.java
package org.techtown.ninepatch;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    // 나인패치란 img 파일 이름 뒤에 9을 붙여 가운데 폭만 늘어나도록해 깨짐 현상이 적게 나타나게 하는 효과이다.

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: 08_AndroidBasic/MyApplication/app/src/main/java/org/techtown/myapplication/MainActivity.kt
package org.techtown.myapplication

import android.app.Activity
import android.os.Bundle
import org.techtown.myapplication.databinding.ActivityMainBinding

class MainActivity : Activity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

    }
}